Formula/python@3.9.rb

```diff
@@ -111,11 +111,14 @@ def install
       --enable-loadable-sqlite-extensions
       --with-openssl=#{Formula["openssl@1.1"].opt_prefix}
       --with-dbmliborder=gdbm:ndbm
+      --enable-optimizations
+      --with-lto
     ]
 
     on_macos do
       args << "--enable-framework=#{frameworks}"
       args << "--with-dtrace"
+      args << "LLVM_AR=/usr/bin/ar"
```

Comment:
This, on the other hand, needs to be reported to the Python developers. If the build fails without it, it's a Python bug. Please add a comment above that line with the Python bug link, so we can check whether it's safe to remove in the future.

Reply:
Sure, I'll get that reported to them and will add the bug link here. Meanwhile if we are going to continue with this workaround until the bug is fixed, is the `/usr/bin` portion here OK? Or is there an expansion I should be using to reach into the CLI tools? Using `xcrun`? I don't know enough about homebrew to know.

Reply:
`/usr/bin` is fine (but is a full path necessary, though?)

Reply:
`LLVM_AR=ar` seems to work fine for me locally.

Reply:
My recollection is that I did need it and your experiment seems to confirm. Do you want me to double check? Or OK to move forward with the explicit path without further work?

Reply:
Filed https://bugs.python.org/issue43109. I'll get the link added to the PR, along with the comment changes requested elsewhere.